test/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Whoa, this is nice. I tend to just use `cd $(dirname $0)`, didn't know about `BASH_SOURCE`.